persistence/src/main/java/cz/muni/fi/pa165/mushrooms/dao/VisitDao.java

```diff
@@ -0,0 +1,53 @@
+package cz.muni.fi.pa165.mushrooms.dao;
+
+import cz.muni.fi.pa165.mushrooms.entity.Visit;
+
+import java.time.LocalDate;
+import java.util.List;
+
+/**
+ *
+ * @author bkompis
+ */
+public interface VisitDao {
+
+    /**
+     * Create a visit.
+     * @param visit the Visit to create in the database.
+     */
+    void create(Visit visit);

```

Comment:
nevadi, ze tam neni PUBLIC ...?

Reply:
Nevadi, interface metody su public by default. Let's keep the conversations in English, please.